README.adoc

```diff
@@ -199,12 +207,13 @@ It's now time to build and run your Arquillian tests.
 Navigate to the `start` directory. First, run the Maven command to package the application.
 Then, run the `liberty-maven-plugin` goals to create the application server, install the features, 
 and deploy the application to the server. The `configure-arquillian` goal configures
-your Arquillian container:
+your Arquillian container. You can learn more about the `configure-arquillian` goal in its
+https://github.com/WASdev/ci.maven/blob/master/docs/configure-arquillian.md[documentation^].
```

Comment:
Suggested update:

You can learn more about this goal in the https://github.com/WASdev/ci.maven/blob/master/docs/configure-arquillian.md[configure-arquillian goal documentation^].

Reply:
updated as suggest